staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -20,16 +20,6 @@ syntax = "proto3";
 package runtime.v1;
 option go_package = "k8s.io/cri-api/pkg/apis/runtime/v1";
 
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
```

Comment:
@HyunSu1768 Firstly, this protobuf file (like most protobuf files) are used in order to autogenerate code. As the comment on line 17 notes, the command `hack/update-codegen.sh` is used to autogenerate code from this file. Any changes to this file should come with the associated changes to the autogenerated code.

Secondly, what makes you say that this import cannot be found and that this code is unused? When I try autogenerating the code, I get no such error. Additionally, autogenerating the code after removing these lines results in different autogenerated code, so it certainly doesn't seem like this is unused.

`gogoproto` is deprecated, so it'd be nice to replace it with something which isn't deprecated, but just removing it entirely isn't the right way to do that.

Reply:
I didn't know that the code was used elsewhere. Can you help me on what direction I should take to resolve this?

Reply:
What exactly are you hoping to accomplish? All this PR does is remove "unused" code, but as far as I can tell this code isn't actually unused (although I'm still curious what led you to believe that in first place- don't take just my word for it!). In that case, since there's no unused code to be removed, there isn't anything to "resolve" and this PR could be closed.

As far as general direction, you may interested in learning more about [protobufs](https://protobuf.dev/) and [this doc explaining how changes to the API work](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md), as those will give you more context on this part of the codebase.

[Here's the repo for gogoproto](https://github.com/gogo/protobuf) which notes that it's deprecated. Like I said, I didn't get an error when I tried generating the code, but if you did then maybe there's a problem to be solved. However, I hadn't heard of this library before I saw your PR, so I can't help with an specific direction there.

Reply:
This code is not unused, so I will close the pr.
Thank you 😁